src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -9,6 +9,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
   <ItemGroup>
     <Compile Include="AllocationHelper.cs" />
+    <Compile Include="ReadOnlyMemory\Strings.cs" />
```

Comment:
nit: move this to the ReadOnlyMemory test ItemGroup.

Reply:
Yup.  Will do.